db/migrate/20220502133047_add_de_feature_flags.rb

```diff
@@ -0,0 +1,23 @@
+class AddDeFeatureFlags < Mongoid::Migration
+  # mirror of FeatureFlag.rb, so this migration won't error if that class is renamed/altered
+  class FeatureFlagMigrator
+    include Mongoid::Document
+    store_in collection: 'feature_flags'
+    field :name, type: String
+    field :default_value, type: Boolean, default: false
+    field :description, type: String
+  end
+  def self.up
+    FeatureFlagMigrator.create!(name: 'differential_expression_frontend',
+                                default_value: false,
+                                description: 'Whether DE UX is enabled')
+    FeatureFlagMigrator.create!(name: 'differential_expression_backend',
```

Comment:
Let's not add a backend feature flag until we have a PR that uses it.  I remember we ended up not really needing the XDS search backend flag

Reply:
OK, will do.  For our shared reference, [here's](https://broadinstitute.slack.com/archives/CBEHTH601/p1651505168662659?thread_ts=1651505162.273389&cid=CBEHTH601) the rationale for two, but I agree the second can wait.

Reply:
Done in 743a1bf.   The new migration is `20220505130728_add_differential_expression_frontend_feature_flag.rb`.

Side note: this was my first time rolling back a Rails migration!  It was simple enough:

```
wm10d-4e7:single_cell_portal_core eweitz$ rake db:migrate:down VERSION=20220502133047_add_de_feature_flags
==  AddDeFeatureFlags: reverting ==============================================
==  AddDeFeatureFlags: reverted (0.1657s) =====================================
```